products/ghc.md

```diff
@@ -0,0 +1,122 @@
+---
+title: GHC
+iconSlug: haskell
+category: lang
+tags: haskell-runtime
+permalink: /ghc
+versionCommand: ghc --version
+releasePolicyLink: https://gitlab.haskell.org/ghc/ghc/-/wikis/GHC-status
+changelogTemplate: https://downloads.haskell.org/ghc/__LATEST__/docs/users_guide/__LATEST__-notes.html
+
+eoasColumn: Further releases planned
+eolColumn: Recommended for use
+releaseDateColumn: true
+
+identifiers:
+- repology: ghc
+- repology: haskell:ghc
+
+auto:
+  methods:
+  - git: https://gitlab.haskell.org/ghc/ghc.git
+    regex: ^ghc-(?P<major>\d+)[.](?P<minor>\d+)[.](?P<patch>\d+)-release$
```

Comment:
Regex looks good: https://regex101.com/r/aRqm9s/2

I'm guessing that the first stable release is always the `.1` patch release?

Reply:
Yes, that guess is correct — no `.0` versions get released — though the project uses terminology differently:
* `x.y.1` where `y` is even — is termed "major release" (may include removals & breaking changes),
* `x.y.z` where `y` is even and `z ≥ 2` — is termed "minor release" (bugfix and perf fixes only).

The `y odd` case is reserved for internal development use (compiler bootstrapping concerns), and aren't ever officially released for GA.

Reply:
... having worded that — adding to explanatory text in-source